plugins/inputs/logfile/logfile.go

```diff
@@ -259,6 +265,10 @@ func (t *LogFile) FindLogSrc() []logs.LogSrc {
 		}
 	}
 
+	
+	t.Log.Infof("Number of file descriptors used by agent / Total allowed file descriptors: %v/%v", t.tailerQueue.Size(), t.tailerQueue.Capacity())
```

Comment:
How many times is FindLogSrc() called? Do we want this at info level?

Reply:
> How many times is FindLogSrc() called?

[1 per second](https://github.com/aws/amazon-cloudwatch-agent/blob/master/logs/logs.go#L93).

>  Do we want this at info level?

Yes. The reason may be in the design doc.
```
Requires the customer to monitor the files on a scheduled interval and can result in missing metrics/logs if not able to handle on time (expected data to be loss in an average of after 30 days)
```
Therefore, would it be good for the customers to detect the issues on time without relying on to restart the agent `with debug flag` (warning is the same level as info). From customer perspective again, would this be corrupt their logs ? The answer is no. I have thought to create another `Ticker`; however, I have thought of create another `time.Ticker` at 1 minute interval but not confidence in it since between log every second and fires a 1 minute interval ticker every second, I would choose the former in terms of memory consumption for customer

Reply:
> Requires the customer to monitor the files on a scheduled interval and can result in missing metrics/logs if not able to handle on time (expected data to be loss in an average of after 30 days)

I don't understand this. Are we telling the customer to monitor something in the CloudWatch agent log file? I thought that the point of this logging was for helping us diagnose. 

---
What does this have to do with memory consumption? I'm asking about if we're willing to accept emitting a log line every second into a file that we do not manage log rotation on. The agent log file can grow unbound, out of the box.

Reply:
> I thought that the point of this logging was for helping us diagnose.

No I believe this is for us and customer. That's why I put it in the `INFO` level instead of `DEBUG` level

>  I'm asking about if we're willing to accept emitting a log line every second into a file that we do not manage log rotation on

Long short answer, yes. However, if you are worrying about `log line every second`, I can manage to using `time.Ticker` to log every 1 minute.